.github/workflows/sycl_update_gpu_driver.yml

```diff
@@ -19,40 +24,14 @@ jobs:
       env:
         BRANCH: ci/update_gpu_driver-linux-${{ env.NEW_DRIVER_VERSION }}
         LLVMBOT_TOKEN: ${{ secrets.LLVM_MAIN_SYNC_BBSYCL_TOKEN }}
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        GITHUB_TOKEN: ${{ secrets.LLVM_MAIN_SYNC_BBSYCL_TOKEN }}
       run: |
         cd $GITHUB_WORKSPACE
         # Set fake identity to fulfil git requirements
         git config --global user.name "GitHub Actions"
         git config --global user.email "actions@github.com"
         git checkout -B $BRANCH
         git add -u
-        git commit -m "[GHA] Uplift GPU RT version for Linux CI" || exit 0   # exit if commit is empty
+        git commit -m "[GHA] Uplift GPU RT version for Linux CI" -m "Uplift GPU RT version for Linux to $NEW_DRIVER_VERSION" || exit 0   # exit if commit is empty
         git push https://$LLVMBOT_TOKEN@github.com/${{ github.repository }} ${BRANCH}
         gh pr create --head $BRANCH --title "[GHA] Uplift GPU RT version for Linux CI" --body "Uplift GPU RT version for Linux to $NEW_DRIVER_VERSION"
```

Comment:
There is explanation in PR description. Let me know if it's not very clear.

We don't use "unstable" anywhere today, except "unstable" containers build. There are 3 areas if we'd like to do a full cleanup (removal) of "unstable" images:

1. Update GPU driver (this specific PR is doing that)
2. dependencies.json
3. sycl_containers.yaml (code you referred to)

I was hesitating on making a full cleanup, made only (1) in this patch limiting patch only to GPU driver update.
As soon as (1) is done (2) and (3) will become effectively a dead code, I expect new unstable builds won't be triggered. I can remove it or can keep it, if we ever want to have some "unstable" containers in the future with some real purpose.

Note that driver version uplift process will keep working through the "latest" images.
In other words after this change:
* https://github.com/intel/llvm/pull/6543 - this will stay
* https://github.com/intel/llvm/pull/6542 - this will stop

Reply:
I don't agree that we should remove "unstable" containers. From time to time the regressions in GPU driver, which block updates. Having unstable container is useful for getting latest status and for those who is not affected by the GPU driver regressions.